tests/bson/bson-utcdatetime-004.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+BSON BSON\UTCDateTime constructor defaults to current time
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$tests = [
+    new MongoDB\BSON\UTCDateTime,
+    new MongoDB\BSON\UTCDateTime(null),
```

Comment:
The current behavior is consistent with your [HHVM implementation](https://github.com/mongodb/mongo-hhvm-driver/blob/73bc3daa7691184a6e091585ad42c01cb965fe41/ext_mongodb.php#L829). Note that DateTime's constructor also defaults to the current time if `null` is provided. I say we keep this behavior and document it.